Author: Martin Pitt <mpitt@debian.org>
Description: Using /tmp for sockets allows everyone to spoof a PostgreSQL server. Thus use /var/run/postgresql/ for "system" clusters which run as 'postgres' (user clusters will still use /tmp). Since system cluster are by far the common case, set it as default.
Forwarded: No, Debian specific configuration with postgresql-common

Index: postgresql-9.0-9.0~beta1/src/include/pg_config_manual.h
===================================================================
--- postgresql-9.0-9.0~beta1.orig/src/include/pg_config_manual.h	2010-01-07 05:53:35.000000000 +0100
+++ postgresql-9.0-9.0~beta1/src/include/pg_config_manual.h	2010-05-01 15:01:17.697230346 +0200
@@ -141,7 +141,7 @@
  * here's where to twiddle it.  You can also override this at runtime
  * with the postmaster's -k switch.
  */
-#define DEFAULT_PGSOCKET_DIR  "/tmp"
+#define DEFAULT_PGSOCKET_DIR  "/var/run/postgresql"
 
 /*
  * The random() function is expected to yield values between 0 and
